build/MSBuildExtensions.targets

```diff
@@ -132,6 +132,13 @@
       <_NETCoreSdkIsPreview Condition=" '$(DropSuffix)' != 'true' ">true</_NETCoreSdkIsPreview>
     </PropertyGroup>
 
+
+    <GetUseBundledNETCoreAppPackageVersionAsDefaultNetCorePatchVersion
```

Comment:
Add a comment here explaining what this task is doing (setting the property to true if patch == 0 and preview == true), and why it is doing it (so that we don't need to manually update the version selection logic between when we ship a final release and when we ship the first patch).

Reply:
Fixed according to the suggestion, and I updated the comment